NineChronicles.Standalone.Executable/Program.cs

```diff
@@ -55,9 +56,15 @@ static void ConfigureSentryOptions(SentryOptions o)
             string host = null,
             [Option('P')]
             ushort? port = null,
+            [Option("swarm-private-key",
+                Description = "The private key used for signing messages and to specify your node. " +
+                              "If you leave this this null, randomly generated value will be used.")]
+            string swarmPrivateKeyString = null,
             [Option('D')]
             int minimumDifficulty = 5000000,
-            [Option("private-key")]
+            [Option("private-key",
```

Comment:
How about using another naming included some prefix, like `mining-private-key` to present its role?

It is a light suggestion.

Reply:
Added in https://github.com/planetarium/NineChronicles.Standalone/pull/199/commits/e9650214b798f9f31f72cb877fa98942440d9d53!